src/live_restore/live_restore_fs.c

```diff
@@ -1435,11 +1459,9 @@ __live_restore_fs_atomic_copy_file(WT_SESSION_IMPL *session, WTI_LIVE_RESTORE_FS
     char *buf = NULL, *source_path = NULL, *dest_path = NULL, *tmp_dest_path = NULL;
     bool dest_closed = false;
 
-    /*
-     * FIXME-WT-14040: We expect that most of these copies will happen prior to the migration phase.
-     * But definitely by the end of the migration this function should not be called. We should
-     * assert that.
-     */
+    WT_ASSERT_ALWAYS(session,
+      __wti_live_restore_get_state(session, lr_fs) == WTI_LIVE_RESTORE_STATE_BACKGROUND_MIGRATION,
```

Comment:
I think this assert is wrong, we could copy them in the "INIT" phase assuming the phase transition is owned by the server. We should only assert that state is not CLEANUP, or COMPLETE.

Reply:
https://github.com/wiredtiger/wiredtiger/pull/11590/commits/e7accc327ae230d77e871da8c444dbdda9561757

The INIT phase is only in the publicly reported live restore state, not the internal state. Now that we've dropped the LOG_COPY stage and export the states in wiredtiger.in perhaps we can unsify the public and private stages into a single enum/set of #defines?